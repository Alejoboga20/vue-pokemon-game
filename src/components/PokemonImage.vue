<template>
	<h3>PokemonImage</h3>
</template>

<script lang="ts">
export default {
	name: 'PokemonImage',
};
</script>

<style lang=""></style>
