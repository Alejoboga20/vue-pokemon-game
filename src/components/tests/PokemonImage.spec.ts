import { shallowMount } from '@vue/test-utils';
import PokemonImage from '../PokemonImage.vue';

describe('PokemonImage', () => {
	test('should match the Snapshot', () => {
		const wrapper = shallowMount(PokemonImage, {
			props: { pokemonId: 1, showPokemon: false },
		});
		expect(wrapper.html()).toMatchSnapshot();
	});

	test('should show the hidden image and pokemon', () => {});
	test('should show the pokemon if showPokemon:true', () => {});
});
