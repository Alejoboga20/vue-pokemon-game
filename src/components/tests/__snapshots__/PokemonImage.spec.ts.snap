// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`PokemonImage > should match the Snapshot 1`] = `
"<div data-v-68dceee4=\\"\\" class=\\"pokemon-container\\"><img data-v-68dceee4=\\"\\" class=\\"hidden-pokemon\\" src=\\"https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/dream-world/1.svg\\" alt=\\"pokemon\\">
  <!--v-if-->
</div>"
`;
